### PR TITLE
Fix of publishing

### PR DESCRIPTION
**Pull request by plokhotnyuk**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -130,14 +135,14 @@ val `javax-annotation` = Def.setting("javax.annotation"      %  "javax.annotatio
 
 lazy val borer = project.in(file("."))
   .aggregate(coreJVM, coreJS)
+  .aggregate(magnoliaJVM, magnoliaJS)
```
[the remark this answers is not in the document]
Comment (sirthias) [reply build.sbt]: Oh, ok. Thank you for spotting and reporting!
Fixed with https://github.com/sirthias/borer/commit/08ab1ed386b8181b30e7b3789d473c6ac932347e...

[the remark this answers is not in the document]
Comment (plokhotnyuk) [reply build.sbt]: Have updated and republished locally - all my changes still required...

Also, found another problem with floats:
```
scala> io.bullet.borer.Json.decode("[0.0]".getBytes).to[Array[Float]].value
res0: Array[Float] = Array(-0.0)
```

Mathias, could you please open Gitter chat for conversations about Borer :)

[the remark this answers is not in the document]
Comment (plokhotnyuk) [reply build.sbt]: I have reset my fork to your latest changes and republished it locally - all my changes still required to complete that successfully....

Also, I have spot another unexpected result when parsing floats:
```
scala> io.bullet.borer.Json.decode("[0.0]".getBytes).to[Array[Float]].value
res0: Array[Float] = Array(-0.0)
```

Mathias, could you please open the Gitter chat for conversations about Borer?

[the remark this answers is not in the document]
Comment (plokhotnyuk) [reply build.sbt]: I have reset my fork to your latest changes and republished it locally - all my changes still required to complete that successfully....

Also, I have spot another unexpected result when parsing floats:
```
scala> io.bullet.borer.Json.decode("[0.0]".getBytes).to[Array[Float]].value
res0: Array[Float] = Array(-0.0)
```

Mathias, could you please open the Gitter chat for conversations about Borer?

**Pull request closed without merging**